councilmatic/settings_jurisdiction.py

```diff
@@ -233,6 +233,119 @@
     },
 ]
 
+BILL_STATUS_DESCRIPTIONS = [
+    {
+        'name': 'ADOPTED',
+        'definition': 'Item was adopted by a majority vote of the Board.',
+        'search_term': 'adopted'
+    },
+    {
+        'name': 'ADOPTED AS AMENDED',
+        'definition': 'Item was adopted as amended by a majority vote of the Board.',
+        'search_term': 'adopted'
+    },
+    {
+        'name': 'APPROVED',
+        'definition': 'Item was approved by a majority vote of the Board.',
+        'search_term': 'approved'
+    },
+    {
+        'name': 'APPROVED AS AMENDED',
+        'definition': 'Item was approved as amended by a majority vote of the Board.',
+        'search_term': 'approved'
+    },
+    {
+        'name': 'APPROVED ON CONSENT CALENDAR',
+        'definition': 'Item was approved on Consent Calendar by a majority vote of the Board.',
+        'search_term': 'approved'
+    },
+    {
+        'name': 'CARRIED OVER --> (Sent To)',
```

Comment:
Does this action actually include `--> (Sent To)`?

Reply:
This is how it was in the spreadsheet Shelly provided. Do you think this is something worth clarifying? There's at least one other place where the board report status name from the spreadsheet doesn't match the code.

Reply:
These should match the statuses as they appear on the site (in our code), IMO.